### ISSUE TITLE
request #76

### ISSUE THREAD

**Issue by mt82** [SBNSoftware/icarus-production#46]
Request | 76
-- | --
Timestamp | 10/9/2024 17:18:43
Email Address | anna.heggestuen@colostate.edu
Name/tag of the request | ICARUS Run2 BNB minbias with RWM timing info
Short description of the sample request | Reprocess all ICARUS run2 BNB minbias files with standard full TPC processing and beam structure information. icaruscode PRs #758 and #751 add the RWM signal timing needed to build the bunch structure to the opFlashes in the calibration ntuples, so a build with these are required
This request involves which detector? | ICARUS
To which working group this request refers to? | OpReco
Contact person Name/Email (in case of issues/questions during the production) | Anna Heggestuen, aheggestcolostate.edu
Intended Deadline | 10/15/2024
Priority (relative to requests submitted in parallel) | 1
Code release version | icaruscode v09_84_00_03 with tarball or >v09_91_02_01
Stage(s) of processing chain to run (workflow) (mark all that apply.) | reco1/Stage0, reco2/Stage1
Total number of events to produce | All of run2 BNB minbias events
Has the workflow been tested? Production will not start unless this answer is 'Yes', please fill the following questions based on the test output (reach out if you need help w/ this step) | Yes
DATA, MC or Other (ex. Overlays)? | DATA
average file size per event | 23 MB (stage0), 47 MB (stage1)
average running time per event | 83 s / event (stage0), 100 s/event (stage1)
average memory usage per event | 3.6 GB (stage0), 3.8 GB (stage1)
gen .fcl |  
g4 .fcl |  
wcsim.fcl (SBND only) |  
detsim .fcl |  
reco/Stage0 .fcl | stage0_run2_icarus.fcl
reco2/Stage1 .fcl | stage1_run2_1d_icarus.fcl
caf .fcl |  
LArCV .fcl |  
Other .fcl (please identify stage, several files can be added in this field if needed) |  
Location of any input files, if needed | /pnfs/sbn/data/sbn_fd/poms_production/data/compression/reconstructed/icaruscode_v09_82_02_01/bnbminbias/compressed_data/
Input dataset, if any | run2_compression_production_v09_82_02_01_bnbminbias_compressed_data
(if the workflow filters) expected filter efficiency (%) |  
Comments - please add any necessary information that was not covered in the other fields | would like to process ICARUS Run2 BNB minBias files with standard TPC processing and beam structure information. icaruscode pull requests #758 and #751 that add the beam timing to the calibration ntuples are merged into develop,  but an official release hasn't been cut yet. we could use icaruscode v09_84_00_03 along with a tarball located here: "/pnfs/icarus/scratch/users/aheggest/tar/local_icaruscode_v09_84_00_beamTiming.tar" or wait for the release after v09_91_02_01 with above PRs included
Stage files to be saveddefault saved files:- ICARUS: Stage1 + Calib tuples + CAFs (FlatCAFs) / LArCV- SBND: Reco2 + Calib tuples + CAFs (FlatCAFs) | reco1/Stage0, reco2/Stage1
